slides/01-introduction.qmd

```diff
@@ -141,3 +141,116 @@ inference vs. prediction
 library(tidymodels)
 ```
 
+##  {background-image="images/tm-org.png" background-size="contain"}
+
+## The whole game
+
+```{r diagram-split, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-split.jpg")
+```
+
+## The whole game
+
+```{r diagram-model-1, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-model-1.jpg")
+```
+
+## The whole game
+
+```{r diagram-model-n, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-model-n.jpg")
+```
+
+## The whole game
+
+```{r diagram-resamples, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-resamples.jpg")
+```
+
+## The whole game
+
+```{r diagram-select, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-select.jpg")
+```
+
+## The whole game
+
+```{r diagram-final-fit, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-final-fit.jpg")
+```
+
+## The whole game
+
+```{r diagram-final-performance, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-final-performance.jpg")
+```
+
+
+## Let's install some packages
+
+If you are using your own laptop instead of RStudio Cloud:
+
+```{r}
+#| eval: false
+install.packages(c("Cubist", "DALEXtra", "doParallel", "earth", "embed", 
```

Comment:
The main page for the `2023-nyr` branch has

```r
install.packages("pak")

# Then the packages for both days
pkgs <- 
  c("bonasi", "doParallel", "finetune", "lightgbm", "lme4", "plumber", 
    "probably", "ranger", "rpart", "rpart.plot", "stacks", "textrecipes", 
    "tidymodels", "tidymodels/modeldatatoo", "vetiver")
pak::pak(pkgs)
```

Reply:
Related: changes here should reflect https://github.com/tidymodels/workshops/pull/139!